src/views/Result.vue

```diff
@@ -1,12 +1,7 @@
 <template>
 	<div class="bg-gray-300 min-h-screen">
 		<ResultHeader
-			:title="isProcessed ? 'Prebieha kontrola originality' : undefined"
-			:description="
-				isProcessed
-					? 'Gratulujeme. Nižšie nájdete podrobné štatistiky kontroly originality vašich prác.'
-					: `Vaše práce sme vzali na spracovanie. Výsledky kontroly originality budú dostupné o chvíľu. Prosím navštívte túto stránku neskôr.`
-			"
+			:description="'Gratulujeme. Nižšie nájdete podrobné štatistiky kontroly originality vašich prác.'"
```

Comment:
The description was previously passed as javascript `:description`, becaouse it contained ternary operator. Now there's no need for it as you only pass simple string.